src/components/Clouds/index.tsx

```diff
@@ -5,8 +5,8 @@ import { CurWeather, IWeatherReducer } from '../../store/types';
 import s from '../Clouds/Clouds.module.scss';
 
 const Clouds = () => {
-	const { weatherCity } = useSelector((state: IWeatherReducer) => state.weathers);
-	const clouds: CurWeather = weatherCity.clouds?.all;
+	const { infoCity } = useSelector((state: IWeatherReducer) => state.weathers);
```

Comment:
infoCity? what info? maybe it's a cityWeatherData?

Reply:
done